src/ui/util/TableConfig.ts

```diff
@@ -0,0 +1,93 @@
+import { idb } from "../../worker/db";
+import getCols, { LeagueVars, MetaCol } from "./columns/getCols";
+import { cloneDeep, uniq } from "lodash-es";
+import { g } from "../../worker/util";
+
+export class TableConfig {
+	public fallback: string[];
+	public columns: MetaCol[];
+	public tableName: string;
+	public vars: LeagueVars | object;
```

Comment:
There are tons of lint errors (including this) if you run `yarn run lint`

Reply:
only a couple hundred

Reply:
I fixed some but many were due to the type Player bindings which we discussed on discord, not done yet but made some progress.